openy_af_vue_app/SMOKE_TESTS.md

```diff
@@ -0,0 +1,368 @@
+3. You should be redirected to the results page
+4. Play with filters by selecting different values
+5. Verify results are updating without reloading the page
+6. Verify soring by different options is working
+
+
+### Expected Results
+
+1. In left sidebar you should see filters:
+- Searchbar by searching by kyewords
+- Expandable filters in different groups (see test case below)
+2. By clicking on any filters the results should be updated with counting of selected filter
+3. On the top of results you should be able to sort results: 
+- Sort by Title (A-Z)
+- Sort by Title (Z-A)
+- Sort by Location (A-Z)
+- Sort by Location (Z-A)
+- Sort by Activity (A-Z)
+- Sort by Activity (Z-A)
+4. All manipulations on the page happens without reloading the page
+
+## Check paragraph settings
+
+### User
+
+Administartor
+
+### Steps
+
+ Limit by category
+1. Log in as admin
+2. Go to existing ""Activity Finder"" page (see test case above) or create a new landing page with paragraphs added (see test case above) 
+3. Click edit on Activity Finder paragraph
+4. In the ""Limit by category"" field select category (for. Group Exercise Classes)
+5. Save node and check the Activity Finder
+
+
+### Expected Results
+
+1. When you visit the Activity Finder Landing page you should be automatically redirected on the first step Ages
+2. All activities should be filtered by selected in the ""Limit by category"" field value
+3. The step of selecting a type of Activity should be skipped
+
+## Check paragraph settings
+
+### User
+
+Administartor
+
+### Steps
+
+Categories exclude
+1. Log in as admin
+2. Go to existing ""Activity Finder"" page (see test case above) or create a new landing page with paragraphs added (see test case above) 
+3. Click edit on Activity Finder paragraph
+4. In the Categories exclude select one or a few categories (for. ex. Group Exercise Classes)
+5. Save node and check the Activity finder
+
+
+### Expected Results
+
+1.In the Activity step, you shouldn't see categories that you excluded
+
+## Check paragraph settings
+
+### User
+
+Administartor
+
+### Steps
+
+Hide Locations selection step
+1. Log in as admin
+2. Go to existing ""Activity Finder"" page (see test case above) or create a new landing page with paragraphs added (see test case above) 
+3. Click edit on Activity Finder paragraph
+4. Check the ""Hide Locations selection step"" checkbox
+5. Save node and check the Activity finder
+
+
+### Expected Results
+
+1. The select location step sould be skipped
+
+## Check paragraph settings
+
+### User
+
+Administartor
+
+### Steps
+
+Hide Program Categories
+1. Log in as admin
+2. Go to existing ""Activity Finder"" page (see test case above) or create a new landing page with paragraphs added (see test case above) 
+3. Click edit on Activity Finder paragraph
+4. Check the ""Hide Program Categories"" checkbox
+5. Save node and check the Activity finder
+
+
+### Expected Results
+
+1. The select location step sould be skipped
```

Comment:
Should be `select program step` instead of `select location step`

Reply:
fixed, tnx